content/en/docs/Tasks/make-index-available-on-cluster.md

```diff
@@ -27,6 +27,11 @@ spec:
       interval: 10m
 ```
 
+### Catalog scoping
+By default, a CatalogSource is scoped to the namespace in which it is created in. For example, a CatalogSource in namespace `A` can only be used in dependency resolution for subscriptions that originate in namespace `A` as well. A subscription in another namespace that references the CatalogSource in `A` will not be able to use the catalog contents for dependency resolution. This provides some basic multitenancy: CatalogSource in one namespace do not interfere with CatalogSource in another. 
+
+However, there is one special namespace, by default the `olm` namespace, which acts as a global catalog namespace. CatalogSources in this namespace can be used for dependency resolution in any other namespace. Installing a catalog in the `olm` namespace will make it available as a subscription target for any namespace on the cluster. The global namespace is configurable via the `OPERATOR_NAMESPACE` environment variable on the OLM operator deployment spec. 
```

Comment:
>The global namespace is configurable via the `OPERATOR_NAMESPACE` environment variable on the OLM operator deployment spec.

Maybe worth extending this sentence with an example. We have to assume readers of this doc do not know how to configure env variable for a deployment.